src/Snapshot/Command/ScheduleSnapshot.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * This file is part of the broadway/snapshotting package.
+ *
+ *  (c) Qandidate.com <opensource@qandidate.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace Broadway\Snapshotting\Snapshot\Command;
+
+class ScheduleSnapshot
+{
+    /**
+     * @var string
+     */
+    private $aggregateRootId;
+
+    /**
+     * @param string $aggregateRootId
+     */
+    public function __construct($aggregateRootId)
```

Comment:
Having gone over the available methods again, I'm starting to have my doubts. Right now we don't have an interface level method to retrieve events up to a certain playhead (unless I've missed something).

In our own workflow we intend to consume this command so we can schedule a job in Resque in order to take snapshots on a different machine, speeding up our request times. I'm pretty sure we will run into this case too.

In the end, snapshots are only used to reconstitute an aggregate quicker. Assuming people aren't  doing weird stuff like somehow removing events, having an even more recent snapshot of your aggregate might be more beneficial.

Maybe I'm not seeing it, but is there a specific pitfall you're worried about?